gtda/mapper/tests/test_nerve.py

```diff
@@ -37,13 +46,8 @@ def test_node_intersection(X):
     assert not any(disjoint_nodes)
 
 
-@settings(deadline=5000)
-@given(X=arrays(dtype=float, unique=True,
-                elements=floats(allow_nan=False,
-                                allow_infinity=False,
-                                min_value=-1e6,
-                                max_value=1e6),
-                shape=array_shapes(min_dims=2, max_dims=2, min_side=11)))
```

Comment:
> Nice refactoring! Though I can't see how it relates to
> 
> > reduced the sizes of examples generated by hypothesis in nerve tests, by lowering the min_size parameter and adding a max_size parameter.
> 
> because I don't see these parameters in the diff.


@ulupo : here, for example.

Reply:
Ah I see!  I guess you meant "min_side" and "max_side" and I was being too rigid :).

Reply:
Ah yes, now I see it! I just meant the sizes, yes